diff --git a/BUILD.gn b/BUILD.gn
index fbdb13a4e..3b450b979 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -225,10 +225,18 @@ group("pdfium_public_headers") {
   ]
 }
 
-component("pdfium") {
+shared_library("pdfium") {
   output_name = "pdfium"
   libs = []
   configs += [ ":pdfium_strict_config" ]
+  if (is_android) {
+    _configs = filter_exclude(configs, [
+      "//build/config/android:hide_all_but_jni_onload",
+      "//build/config/android:hide_all_but_jni",
+    ])
+    configs = []
+    configs = _configs
+  }
   public_configs = [ ":pdfium_public_config" ]
 
   deps = [
@@ -263,6 +269,9 @@ component("pdfium") {
       "gdi32.lib",
       "user32.lib",
     ]
+    sources = [
+      "resources.rc",
+    ]
   }
 
   if (is_mac) {
